docs/source/transcript_selection.rst

```diff
@@ -0,0 +1,35 @@
+.. _transcript_selection_policy:
+
+Transcript Selection
+====================
+
+One of the core uses of Cool-Seq-Tool is to acquire and use consensus-based, representative transcripts in performing genomic analysis. Here, we describe the selection processes, programmed in the :py:class:`MANETranscript <cool_seq_tool.mappers.mane_transcript.MANETranscript>` class, for choosing the best available transcripts that are compatible with requested data.
+
+We rely heavily on transcripts annotated under the `Matched Annotation from NCBI and EMBL-EBI (MANE)` Transcripts project. For more information on the MANE project, see the `NCBI MANE page <https://www.ncbi.nlm.nih.gov/refseq/MANE/>`_.
+
+.. _transcript_compatibility:
+
+Transcript compatibility
+------------------------
+
+The following validation checks are performed to determine compatibility of a transcript position:
+
+* The position exists on an accession
+* "Validating reference sequences"  # TODO what does this mean?
+* "Validating exon structure"  # TODO what does this mean?
```

Comment:
@korikuzma can you clarify these two lines? They're taken from the previous markdown docs file

Reply:
IIRC validating reference sequences means ensuring actual reference sequence matches expected reference sequence for a given ac/pos. Exon structure has to deal with ensuring exon numbers are the same.

Reply:
🙌